src/main/java/net/snowflake/ingest/streaming/internal/ParquetFlusher.java

```diff
@@ -48,12 +48,12 @@ public ParquetFlusher(
 
   @Override
   public SerializationResult serialize(
-      List<ChannelData<ParquetChunkData>> channelsDataPerTable, String filePath)
+      List<ChannelData<ParquetChunkData>> channelsDataPerTable, String filePath, int chunkIndex)
       throws IOException {
     if (enableParquetInternalBuffering) {
       return serializeFromParquetWriteBuffers(channelsDataPerTable, filePath);
```

Comment:
Why don't we update the `PRIMARY_FILE_ID_KEY` in the `serializeFromParquetWriteBuffers` path?

Reply:
Nvm, I see that Toby has already commented that this is unused.

Reply:
Right, and here is the PR https://github.com/snowflakedb/snowflake-ingest-java/pull/829 to clean that up.